src/components/regionalization/RegionalizationBar/RegionalizationBar.tsx

```diff
@@ -2,26 +2,26 @@ import type { HTMLAttributes } from 'react'
 import { useModal } from 'src/sdk/ui/modal/Provider'
 import Button from 'src/components/ui/Button'
 import Icon from 'src/components/ui/Icon'
+import { useSession } from '@faststore/sdk'
 
 interface RegionalizationBarProps extends HTMLAttributes<HTMLDivElement> {
-  content?: string
   classes: string
 }
 
 export default function RegionalizationBar({
-  content,
   classes,
   ...otherProps
 }: RegionalizationBarProps) {
   const { setIsRegionalizationModalOpen } = useModal()
+  const { postalCode } = useSession()
 
   return (
     <div data-fs-regionalization-bar className={classes} {...otherProps}>
       <Button onClick={() => setIsRegionalizationModalOpen(true)}>
         <Icon name="MapPin" width={24} height={24} />
-        {content ? (
+        {postalCode ? (
           <>
-            <span>{content}</span>
+            <span>{postalCode}</span>
```

Comment:
Add key to this span and the other below that has "set your location".

Reply:
Could you explain why the key is needed here? I know it's normally used when rendering an array of items, but know very little of other use cases.

I found this other use case (https://kentcdodds.com/blog/understanding-reacts-key-prop), but not sure it's the same here.

Reply:
Thanks for bringing this up ☺️

Reply:
Yeah, i was wrong asking to add keys, didn't make sense